patroni/postgresql/sync.py

```diff
@@ -201,35 +221,35 @@ def _handle_synchronous_standby_names_change(self) -> None:
 END;$$""")
         self._postgresql.reset_cluster_info_state(None)  # Reset internal cache to query fresh values
 
-    def current_state(self, cluster: Cluster) -> Tuple[CaseInsensitiveSet, CaseInsensitiveSet]:
-        """Finds best candidates to be the synchronous standbys.
+    def _get_replica_list(self, cluster: Cluster) -> Iterator[Tuple[int, str, str, int, bool]]:
+        """Yields candidates based on higher replay/remote_write/flush lsn.
 
-        Current synchronous standby is always preferred, unless it has disconnected or does not want to be a
-        synchronous standby any longer.
+        .. note::
+            Tuples are reverse ordered by sync_state and LSN fields so nodes that already synchronous or having
```

Comment:
```suggestion
            Tuples are reverse ordered by ``sync_state`` and LSN fields so nodes that already synchronous or having
```

Reply:
Do you think it is worth it mentioning that nodes with `nosync` tag are not considered as the idea of the method is to get a list of replicas that can be synchronous?